docs/reference/troubleshooting.md

```diff
@@ -0,0 +1,235 @@
+# Troubleshooting
+
+This guide covers the most common issues you may encounter when using `conda-pypi` and how to resolve them.
+
+## Environment Issues
+
+### Missing Python or pip requirements
+
+**Problem**: `conda-pypi` fails with an error about missing Python or pip.
+
+**Error messages**:
+```
+Target environment at /path/to/env requires python>=3.2
+Target environment at /path/to/env requires pip>=23.0.1
+```
+
+**Solution**: Ensure your target environment has the required dependencies:
+
+```bash
+# Install required dependencies in your environment
+conda install -n myenv python>=3.9 pip>=23.0.1
+
+# Or when creating a new environment
+conda create -n myenv python>=3.10 pip
+```
+
+### Invalid environment
+
+**Problem**: Commands fail when specifying a non-existent environment.
+
+**Error messages**:
+- `environment does not exist`
+- `python>=3.2 not found`
+
+**Solution**:
+```bash
+# Check if environment exists
+conda env list
+
+# Create the environment if it doesn't exist
+conda create -n myenv python=3.10 pip
+
+# Use correct environment name or path
+conda pypi install -n myenv package-name
+```
+
+### Permission errors
+
+**Problem**: Cannot write to environment directory.
+
+**Solutions**:
+```bash
+# Check environment permissions
+ls -la $(conda info --base)/envs/
+
+# Fix ownership if needed (Linux/macOS)
+sudo chown -R $USER $(conda info --base)/envs/myenv
+
+# On Windows, run conda prompt as administrator
+```
+
+## Package Resolution Issues
+
+### Package not found on PyPI
+
+**Problem**: Package doesn't exist or has a different name on PyPI.
+
+**Error messages**:
+- `No matching distribution found`
+- `Could not find a version that satisfies the requirement`
+- `404 Client Error`
+
+**Solutions**:
+```bash
+# Check package name on PyPI
+pip search package-name  # or visit pypi.org
+
+# Try common name variations
+conda pypi install python-package-name  # instead of package-name
+conda pypi install package_name         # underscores instead of dashes
+
+# Use exact PyPI name
+conda pypi install "Package-Name"       # match exact case
```

Comment:
Aren't we normalizing on input? I think we do when we map packages too (I recall using `.lower()` and `.replace()`).

Reply:
Good point, I removed the exact match and underscore replacement sections.